setup.cfg

```diff
@@ -1,2 +0,0 @@
-[wheel]
-universal = 1
```

Comment:
Are we still marking this somehow? I don't see it in `pyproject.toml`.

Reply:
Its an implication of `requires-python = ">=2.7,<3.9,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"`
If you run `flit build` you'll see a universal wheel gets plopped to dist/. There is no tox target for this, just `tox -e publish`, but that's safe to run since pypi does not allow over-writes.